File: ArxivExpress/ArxivExpress/Features/ArticleList/Contributor.cs
namespace ArxivExpress.Features.ArticleList
{
    public class Contributor
    {
        private string _name;
        private string _email;

        public string Email { get { return _email ?? "unknown"; } }
        public string Name { get { return _name ?? "unknown"; } }

        public Contributor(string name, string email)
        {
            _name = name;
            _email = email;
        }
    }
}

File: ArxivExpressFlutter/lib/shared/models/contributor.dart
// ****************************************************************************
//    File "contributor.dart"
//    Copyright © Dmitry Morozov 2025
//    Flutter reimplementation of ArxivExpress Xamarin app
// ****************************************************************************

class Contributor {
  final String name;
  final String? email;

  Contributor({
    required this.name,
    this.email,
  });

  factory Contributor.fromXml(Map<String, dynamic> contributorData) {
    return Contributor(
      name: contributorData['name'] ?? '',
      email: contributorData['email'],
    );
  }
}